src/components/Tooltip/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import FDS from 'lib/dictionary/js/styleConstants';
-import Tippy from '@tippyjs/react';
+import LazyTippy from 'components/Popover/LazyTippy';
```

Comment:
We should probably make some `private` folder to put common abstractions between components, while at the same time communicating that people should absolutely not import them, but that probably requires discussion and strategizing that may blow out the scope here. I just decided it would be easier for LazyTippy to live in `Popover` and we can come back and do an organization ticket for the rest.